library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -231,6 +232,23 @@ public void onMapClick(LatLng latLng) {
     polyline.removeFromGoogleMap();
   }
 
+  @Override
+  public void zoomToFitMarkers(int padding, AirMapMarker... markers) {
+    LatLngBounds.Builder b = new LatLngBounds.Builder();
+    for(AirMapMarker marker : markers){
+      b.include(marker.getLatLng());
+    }
+    LatLngBounds bounds = b.build();
+    final CameraUpdate cu = CameraUpdateFactory
+            .newLatLngBounds(bounds, padding);
+    googleMap.setOnMapLoadedCallback(new GoogleMap.OnMapLoadedCallback() {
```

Comment:
if the map is already loaded, this is not necessary to add this loaded callback. also, this will overwrite any map loaded callback that may have been set previously